aten/src/ATen/native/sparse/cuda/SparseCsrTensorMath.cu

```diff
@@ -100,6 +105,35 @@ void convert_indices_from_csr_to_coo_cuda(const Tensor& indices, const Tensor& c
   C10_CUDA_KERNEL_LAUNCH_CHECK();
 }
 
+template <typename scalar_t, typename value_t>
+void _apply_sparse_csr_lu_solve(
+  const at::sparse_csr::SparseCsrTensor& input,
```

Comment:
Are you aware that `at::sparse_csr::SparseCsrTensor` does nothing? It's just an alias to `Tensor`.